ci/Dockerfile.photon

```diff
@@ -1,17 +1,16 @@
 FROM photon:latest
 
-MAINTAINER csiddharth@vmware.com
-
-RUN tdnf update  -q -y
-RUN tdnf remove  -q -y toybox
-RUN tdnf install -q -y --enablerepo=photon-debuginfo \
+RUN tdnf update  -qy
+RUN tdnf remove  -qy toybox
+RUN tdnf install -qy --enablerepo=photon-debuginfo \
```

Comment:
I think (while you are at it), we should remove the `-q` flag. I don't see why we shouldn't see the progress and it may make it easier to debug if there are issues.

Reply:
Done